js/_website/src/lib/templates/gradio/04_helpers/11_examples.svx

```diff
@@ -63,6 +63,31 @@ None
 <ParamTable parameters={obj.attributes} />
 
 
+### Examples
```

Comment:
If you update the examples, you don't update the cache right? Would be good to make a note of that